contrib/helm-charts/portus/templates/portus-deployment.yaml

```diff
@@ -30,11 +30,14 @@ spec:
           ports:
             - containerPort: {{ .Values.portus.service.port }}
           livenessProbe:
-            exec:
-              command:
-                - /api/v1/health
-            initialDelaySeconds: 240
-            periodSeconds: 5
+            httpGet:
+              {{- if .Values.portus.tls.enabled }}
+              scheme: HTTPS
+              {{- end }}
+              path: /api/v1/health
+              port: {{ .Values.portus.service.port }}
+            initialDelaySeconds: 300
```

Comment:
Were you seeing failures with the lower timeout values? 5 minutes and 10 seconds respectively seem too long.

Reply:
As I measured it just now, Portus took about 7 seconds to return the response in my environment.
```
$ curl https://registry.example.com/api/v1/health -s -o /dev/null -w "Total: %{time_total}\n"
Total: 6.711
```
I will need to validate the proper value of `initialDelaySeconds`. I will work on it later.